coralproject/talk
# [CORL-1242] Ensure SSO Access Tokens are not persisted

## What does this PR do?

This ensures that access tokens provided to the stream for single sign on are not persisted after navigation.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Provide an access token via the `embed.login(<access token>)` method, and refresh the page. See that you are logged out after the refresh.
